### ISSUE TITLE
Bless does not appears inside client

### ISSUE THREAD

**Issue by lucasgrizante** [otland/forgottenserver#3974]
### Before creating an issue, please ensure:
- [X] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/support.16/ for support)
- [X] This issue is reproducible without changes to the C++ code in this repository

### Steps to reproduce (include any configuration/script required to reproduce)
1. Buy all blesses from NPC

### Expected behaviour
It should appear that I have X blesses

### Actual behaviour
![image](https://user-images.githubusercontent.com/40500081/155304067-0e59f95b-3afc-4b92-9066-75bed6ca042e.png)

### Environment
TFS master branch, Tibia 12.85

**Comment by EPuncker** [comment]
yeah we are missing lots of feature from 11/12 version, this is a known issue among with others #3871
